### ISSUE TITLE
error on cpSync

### ISSUE THREAD

**Issue by winniezzz** [tobua/create-react-native-plugin#3]
bun create react-native-plugin myplugin
31 |
32 | const packagePath = dirname(new URL(import.meta.url).pathname)
33 | const templateDirectory = join(packagePath, 'template')
34 | const destinationDirectory = join(process.cwd(), name.regular)
35 |
36 | cpSync(templateDirectory, destinationDirectory, { recursive: true })
     ^
ENOENT: No such file or directory
   errno: -2
 syscall: "copyfile"

npm version 7.20.6
node version 18.14.0

**Comment by tobua** [comment]
Hey @winniezzz 

Thanks for trying out the plugin! I'm not experiencing any issues downloading the template myself. I've recently updated it to use a more modern approach and backwards compatibility isn't a priority here and it will only work with Bun.

Make sure to update Bun to the current "1.1.27" (for `bun pm pack`) and Node.js to at least the LTS "v20.17.0". Although, the plugin shouldn't be falling back to Node.js in the first place, so node doesn't really matter.

```ts
const packagePath = dirname(new URL(import.meta.url).pathname)
```

Due to an older version there are probably issues resolving this to the path of the node_module where the template contents are taken from. I've tested it only on macOS, which OS are you using? That might be an issue as well.

Make sure to use the latest version when trying again with `bun create react-native-plugin@latest myplugin`.
